tmk_core/protocol/lufa/adafruit_ble.cpp

```diff
@@ -10,6 +10,10 @@
 #include "action_util.h"
 #include "ringbuffer.hpp"
 #include <string.h>
+#define SAMPLE_BATTERY
+#ifdef SAMPLE_BATTERY
```

Comment:
The ifdef here is not necessary, actually, and the define can go back where it was.

Reply:
Done.